cmd/openshift-tests/e2e.go

```diff
@@ -85,6 +85,8 @@ var staticSuites = testSuites{
 				}
 				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
 			},
+			// doubling default test timeout (15 mins) which is needed for external ip configuration related serial tests.
+			TestTimeout:         30 * time.Minute,
```

Comment:
This is a little troubling, doubling the timeout globally could push us past the time limit for CI jobs. Certain we need more than an already pretty crazy 15 minutes?

Reply:
Yes @dgoodwin . But It looks `TestTimeout` can be set only at test suite level and not at test level. Should we move these two external ip tests into a new test suite to configure with 30 mins timeout ?

Reply:
Lets see what approver thinks.

Reply:
If the increased timeout causes a problem, we might have to evaluate putting these longer running network tests in their own suite, but let's try it for now